Update recipe/recipe.yaml for v2.13.0rc4

jlenh opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

jlenh committed:
Update meta.yaml

jlenh commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

Commit:
MNT: Re-rendered with conda-smithy 3.52.2 and conda-forge-pinning 2025.09.23.08.02.1

jlenh commented:
Skipping #91 for this PR (this should be included in this new release candidate @valeriupredoi https://github.com/conda-forge/esmvalcore-feedstock/pull/91#discussion_r2344157554)

bouweandela commented:
It would be nice to include the changes from https://github.com/conda-forge/esmvalcore-feedstock/pull/90#issuecomment-3285300181 here

jlenh commented:
> It would be nice to include the changes from https://github.com/conda-forge/esmvalcore-feedstock/pull/90#issuecomment-3285300181 here

Oh my bad, I thought it was already there since it was merged (but it's in the main branch)!
I'll adapt that

jlenh committed:
Update feedstock structure w/ recipe.yaml

jlenh commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to rerender for you but ran into some issues. Please check the [output logs](https://github.com/conda-forge/conda-forge-webservices/actions/runs/17944546349) of the GitHub Actions workflow below for more details. You can also ping `conda-forge/core` (using the `@` notation) for further assistance or you can try [rerendering locally](https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-with-conda-smithy-locally).


The following suggestions might help debug any issues:
* Is the `recipe/{meta.yaml,recipe.yaml}` file valid?
* If there is a `recipe/conda-build-config.yaml` file in the feedstock make sure that it is compatible with the current [global pinnnings](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml).
* Is the fork used for this PR on an organization or user GitHub account? Automated rerendering via the webservices admin bot only works for user GitHub accounts.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/17944546349](https://github.com/conda-forge/conda-forge-webservices/actions/runs/17944546349). Examine the logs at this URL for more detail.</sub>

jlenh committed:
Update conda-forge.yml

jlenh commented:
> > It would be nice to include the changes from [#90 (comment)](https://github.com/conda-forge/esmvalcore-feedstock/pull/90#issuecomment-3285300181) here
> 
> Oh my bad, I thought it was already there since it was merged (but it's in the main branch)! I'll adapt that

I guess the build is exiting on an error because the template is not updated for the `recipe.yaml` format @bouweandela ?

bouweandela commented:
I can have a look tomorrow if needed. Did you try the suggestion of rendering locally?

valeriupredoi commented:
I'm on station, can help out here :beer:

jlenh committed:
MNT: Re-rendered with conda-smithy 3.52.2 and conda-forge-pinning 2025.09.23.08.02.18

Other tools:
- conda-build 25.7.0
- rattler-build 0.47.0
- rattler-build-conda-compat 1.4.6

valeriupredoi commented:
@jlenh let me try a test PR off my fork, I took a look at your fork and it's rather divergent (140+ commits ahead of, 42 commits behind `main`)

jlenh commented:
> @jlenh let me try a test PR off my fork, I took a look at your fork and it's rather divergent (140+ commits ahead of, 42 commits behind `main`)

I was kind of wondering about that between the `main` and `rc` branches indeed.

I re-rendered locally and it seems to work, let's see about the build of the feedstock here!

valeriupredoi commented:
excellent! Keeping an eye on the build test myself - we shouldn't rerender locally everytime we release though, guessing a few of those files were not picked up in the `main` to here merge

valeriupredoi commented:
super! All built - and fast too! Indeed the migration done in https://github.com/conda-forge/esmvalcore-feedstock/pull/92/commits/846bcae78cbafcc26487f5bc5c0c4495dd2dca5c contained only the changes to the recipe

valeriupredoi commented:
@bouweandela does this need rerendering now too, by the regro bot, here? Methinks yes

jlenh commented:
> super! All built - and fast too! Indeed the migration done in [846bcae](https://github.com/conda-forge/esmvalcore-feedstock/commit/846bcae78cbafcc26487f5bc5c0c4495dd2dca5c) contained only the changes to the recipe

Oh yeah it shaved off at least ~2min on the built. And indeed that might have been the issue on the previous commits.

jlenh commented:
> @bouweandela does this need rerendering now too, by the regro bot, here? Methinks yes

Should I request the bot for a re-render again then?

valeriupredoi commented:
yeah I think so, let's test all the usual steps one would take when releasing, what can possibly go wrong :bomb:

jlenh commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to rerender for you, but it looks like there was nothing to do.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/17947017073](https://github.com/conda-forge/conda-forge-webservices/actions/runs/17947017073). Examine the logs at this URL for more detail.</sub>

jlenh commented:
> yeah I think so, let's test all the usual steps one would take when releasing, what can possibly go wrong 💣

Well that looks good when it happens there is nothing to do I guess :sweat_smile:

valeriupredoi commented:
> Hi! This is the friendly automated conda-forge-webservice.
> 
> I tried to rerender for you, but it looks like there was nothing to do.
> 
> This message was generated by GitHub Actions workflow run https://github.com/conda-forge/conda-forge-webservices/actions/runs/17947017073. Examine the logs at this URL for more detail.

perfect, as it should be :balloon:

valeriupredoi reviewed (approved):
many thanks @jlenh and @bouweandela :beers:

jlenh commented:
> many thanks @jlenh and @bouweandela 🍻

Good to merge I guess then?

valeriupredoi commented:
we have liftoff of STS-2.13rc4 :grin:
